hips/draw/simple.py

```diff
@@ -202,18 +155,12 @@ def make_sky_image(geometry: WCSGeometry, hips_survey: HipsSurveyProperties, til
 
     Returns
     -------
-    data : `~numpy.ndarray`
-        Output image pixels
+    painter : `SimpleTilePainter`
```

Comment:
For now, I would suggest to only return `data` like we did before.

I think the Painter should be more or less an implementation detail, and we should introduce a results class in the future that is then uniform for different painters and presents the result info in a better way than a painter class with all the methods mixed in.